include/aws/common/ring_buffer.h

```diff
@@ -17,6 +17,19 @@
 
 #include <aws/common/atomics.h>
 
+#ifdef CBMC
+#    define AWS_ATOMIC_LOAD_PTR(dest_ptr, ring_buf, atomic_ptr)                                                        \
+        dest_ptr = aws_atomic_load_ptr(atomic_ptr);                                                                    \
+        assert(__CPROVER_POINTER_OBJECT(dest_ptr) == __CPROVER_POINTER_OBJECT(ring_buf->allocation));                  \
+        assert(aws_ring_buffer_check_atomic_ptr(ring_buf, dest_ptr));
+#    define AWS_ATOMIC_STORE_PTR(src_ptr, ring_buf, atomic_ptr)                                                        \
```

Comment:
The parameters to this macro are swapped compared with `aws_atomic_store_ptr` (where `src_ptr` comes after `atomic_ptr`). This is confusing and it seems that it would be easy for people to make mistakes when uting it. Is there a good reason why you chose this order?

Reply:
Done!